script.js

```diff
@@ -84,3 +97,76 @@ function submitAddTaskForm() {
   document.getElementById("titleInput").value = "";
   return false;
 }
+
+// onclick function to blur -ishaan
+function changeClass(element) {
```

Comment:
change this to toggleDoneClass (you should make it so that you dont need to remove the todo class)